tests/crypto/readfile.h

```diff
@@ -22,6 +22,10 @@
 #define max_date_elements 6
 #define max_coordinates_size 32
 
+#ifdef _MSC_VER
```

Comment:
I am not sure but I think this should check _WIN32 instead of _MSC_VER since I think sscanf_s is platform specific not compiler specific.

Reply:
If I change to _WIN32 then there will be a C99/C11 compilation problem.

Reply:
Ok thanks for checking.  Resolving this comment.